google/cloud/storage/tests/bucket_integration_test.cc

```diff
@@ -465,37 +465,47 @@ TEST_F(BucketIntegrationTest, NotificationsCRUD) {
       client.CreateBucketForProject(bucket_name, project_id, BucketMetadata());
 
   auto current_notifications = client.ListNotifications(bucket_name);
-  EXPECT_TRUE(current_notifications.empty())
+  ASSERT_TRUE(current_notifications.ok())
```

Comment:
Not crucial for this PR, but do you think we want an `EXPECT_OK(status)` (and assert)? It would always print the failing Status object if the assertion fails.

Reply:
You mean create one?  That would be nice indeed.